### ISSUE TITLE
Revamp app size

### ISSUE THREAD

**Issue by NyuBlara** [scrime-u-bordeaux/web-midifile-performer#118]
Here's a snippet of what the app looks like when score-width is 70vw and the app header is removed :

![image](https://github.com/user-attachments/assets/78def3e3-0640-4e58-ac17-5fcefe730a03)

This calls for two things : 

- Widening the score width, 
- Adding an option to collapse the header (go "full-screen")

**Comment by josephlarralde** [comment]
Yes, a GUI remix is necessary, but this goes beyond the display size of the score.
I think we should also get rid of the first-steps page and replace it by a default loaded musicxml score.
Also, some tips should appear when the users start the app, except if they have disabled "show tips at startup" in a popup window. Users report that it is still not totally clear how to play, and expecting them to go have a look at the help page by themselves seems a bit optimistic.
Finally, the menu should be reworked too (probably removed and its content disseminated in the GUI), and the GUI should allow simultaneous access to configuration parameters and performance.

**Comment by NyuBlara** [comment]
I agree with all of this, but I believe the two steps I outlined would be a good start.

**Comment by josephlarralde** [comment]
Indeed !

**Comment by NyuBlara** [comment]
This will be harder now that #119 has been implemented...

**Comment by josephlarralde** [comment]
Now that #152 has been closed, we don't have a first steps page anymore.
It is getting very annoying that whenever we visit the Doc and About tabs then come back to the App the score gets loaded again. This feels like a waste of resources. I would like to include MidifilePerformer.vue in App.vue and have everything else as overlays (maybe still managed by a vue-router) so that the score never needs to be reloaded as long as the user does't refresh the page.

**Comment by josephlarralde** [comment]
On small screens, a collapsible menu sounds desirable since sometimes the score can be completely hidden between the menu and the (newly) sticky controls footer. Maybe we could also have collapsible or draggable controls ?

And a wider score too of course. IMO its size should be a percentage of the window width. I'd really love to have a nice responsive layout ready to build mobile features upon like touch control.
